cluster/gce/configure-vm.sh

```diff
@@ -284,6 +284,7 @@ opencontrail_public_subnet: '$(echo "$OPENCONTRAIL_PUBLIC_SUBNET")'
 enable_manifest_url: '$(echo "$ENABLE_MANIFEST_URL" | sed -e "s/'/''/g")'
 manifest_url: '$(echo "$MANIFEST_URL" | sed -e "s/'/''/g")'
 manifest_url_header: '$(echo "$MANIFEST_URL_HEADER" | sed -e "s/'/''/g")'
+kubelet_port: '$(echo "$KUBELET_PORT")'
```

Comment:
There're other lines without a sed that doubles single quotes. Here we're passing a number (we'll fail sooner if it's not), so there's no need for it.